.github/workflows/build.yml

```diff
@@ -8,30 +8,38 @@ on:
     
 jobs:
   tests:
-    runs-on: macos-12
+    runs-on: ${{ matrix.os }}
     continue-on-error: false
     strategy:
       fail-fast: false
       matrix:
         include:
         
+        - version: '16.4'
+          displayname: 'iPhone 14'
+          os: 'macos-13-xl'
```

Comment:
```suggestion
          os: 'macos-13-xlarge'
```

Reply:
Interesting one - would create a PR that does some more fixes to the `build.yml` and test it in isolation 👌